test/cql-pytest/test_range_and_slice.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2021-present ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+
+#############################################################################
+# Tests the calculation of partition range and slice.
```

Comment:
Nitpick: I'm not sure this new file "test_range_and_slice.py" is a great choice, it's not clear (at least not to me) what tests would belong in this file, and why even this one does (it's a test about DELETE, and about IN - what makes it a "range and slice" test?).
But we can always move tests around later.